src/pages/RouteDefinitions.tsx

```diff
@@ -139,7 +139,11 @@ export const routes: RouteDefinition[] = [
   }),
   createRouteDefinition({
     path: 'explore/pools/:chainName/:poolAddress',
-    getElement: () => <PoolDetails />,
+    getElement: () => (
+      <Suspense fallback={null}>
```

Comment:
can this Suspense wrapper go inside PoolDetails?

Reply:
No it needs to be top level or else Suspense will show the default pink loading circle. You can see this pattern used throughout the RouteDefinitions